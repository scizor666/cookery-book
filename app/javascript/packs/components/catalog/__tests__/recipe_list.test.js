import React from 'react';
import renderer from 'react-test-renderer';
import {recipes} from './test_data';
import RecipeList from '../recipe_list';

jest.mock('../recipe_card', () => 'RecipeCard');

test('display a recipe list', () => {
    const wrapper = renderer.create(
        <RecipeList recipes={recipes}/>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
});