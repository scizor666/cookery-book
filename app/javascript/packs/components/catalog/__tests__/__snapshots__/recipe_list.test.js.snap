// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display a recipe list 1`] = `
<div
  className="row"
>
  <div
    className="col-md-4 mt-2"
  >
    <RecipeCard
      placeholderUrl="/placeholder-320x240.png"
      recipe={
        Object {
          "description": "some description here",
          "id": 123456,
          "image_url": "http://some.url/with_picture.png",
          "name": "some recipe",
          "short_description": "some short description here",
        }
      }
    />
  </div>
  <div
    className="col-md-4 mt-2"
  >
    <RecipeCard
      placeholderUrl="/placeholder-320x240.png"
      recipe={
        Object {
          "description": "some other description here",
          "id": 678990,
          "name": "some other recipe",
          "short_description": "some other short description here",
        }
      }
    />
  </div>
</div>
`;
